snark-verifier/src/pcs.rs

```diff
@@ -73,34 +101,46 @@ where
     where
         T: TranscriptRead<C, L>;
 
+    /// Verify old [`AccumulationScheme::Accumulator`]s are accumulated properly
+    /// into a new one with the [`AccumulationScheme::Proof`], and returns the
+    /// new one as output.
     fn verify(
         vk: &Self::VerifyingKey,
         instances: &[Self::Accumulator],
         proof: &Self::Proof,
     ) -> Result<Self::Accumulator, Error>;
 }
 
+/// Accumulation scheme decider.
 pub trait AccumulationDecider<C, L>: AccumulationScheme<C, L>
 where
     C: CurveAffine,
     L: Loader<C>,
 {
+    /// Deciding key.
```

Comment:
Sure will add it. So it's the key for decider to perform `decide` function, it might contain different stuff than verifying key, since it's trying to decide if an accumulator is correct or not.

For instance, in atomic accumulation scheme for KZG, the deciding key is 2 G2 elements used in PLONK.